scripts/crons/crontabber.sh

```diff
@@ -8,12 +8,12 @@
 
 NAME=`basename $0 .sh`
 lock --ignore-existing $NAME
-CMD="${PYTHON} ${APPDIR}/socorro/cron/crontabber_app.py"
+export CMD="${PYTHON} ${APPDIR}/socorro/cron/crontabber_app.py"
 LOG=/var/log/socorro/crontabber.log
```

Comment:
Nit, but when file testing in bash, one should generally use `-f` and `-r` together - it'll save you headaches down the road. :smile: 

```bash
CT_INI="/etc/socorro/crontabber.ini"
if [ -f $CT_INI ] && [ -r $CT_INI ]; then
    $CMD --admin.conf=${CT_INI} >> $LOG 2>&1
```